src/jackdaw/serdes/avro.clj

```diff
@@ -671,7 +674,8 @@
    {:keys [avro/schema
            avro/coercion-cache
            key?
-           deserializer-properties]
+           deserializer-properties
```

Comment:
What is this?

Reply:
it pre-exists this pr, introduced in commt #8551c18c836b7b52e0a0b16b9b041d30b3eb4a0c